app/api/views.py

```diff
@@ -146,6 +148,8 @@ def publish_metadata_record(_oid):
     with open(save_path, 'w+') as f:
         f.write(iso)
 
```

Comment:
add logic to see if hostname contains `localhost`.  If not, gptInsert, if so, skip.